src/qt/recentrequeststablemodel.cpp

```diff
@@ -125,12 +125,10 @@ void RecentRequestsTableModel::updateAmountColumnTitle()
 /** Gets title for amount column including current display unit if optionsModel reference available. */
 QString RecentRequestsTableModel::getAmountTitle()
 {
-    QString amountTitle = tr("Amount");
     if (this->walletModel->getOptionsModel() != NULL)
-    {
-        amountTitle += " ("+BitcoinUnits::name(this->walletModel->getOptionsModel()->getDisplayUnit()) + ")";
-    }
-    return amountTitle;
+        return tr("Requested") + " ("+BitcoinUnits::name(this->walletModel->getOptionsModel()->getDisplayUnit()) + ")";
+    else
+        return tr("Requested amount");
```

Comment:
I don't think we need a special translation for the case that the model is not set, it is an error case which shouldn't crash but there's no need to make translators work for it.